doc/classes/Node2D.xml

```diff
@@ -96,44 +97,46 @@
+			Local position (translation) of this node in local space (relative to the parent node). This is equivalent to the [member transform]'s [member Transform2D.origin]. See also [member global_position].
 		</member>
 		<member name="rotation" type="float" setter="set_rotation" getter="get_rotation" default="0.0">
-			Rotation in radians, relative to the node's parent. See also [member global_rotation].
-			[b]Note:[/b] This property is edited in the inspector in degrees. If you want to use degrees in a script, use [member rotation_degrees].
+			Local rotation of this node in radians, in local space (relative to the parent node). See also [member global_rotation].
+			[b]Note:[/b] This property is edited in degrees in the inspector. If you want to use degrees in a script, use [member rotation_degrees].
 		</member>
 		<member name="rotation_degrees" type="float" setter="set_rotation_degrees" getter="get_rotation_degrees">
-			Helper property to access [member rotation] in degrees instead of radians. See also [member global_rotation_degrees].
+			The [member rotation] of this node, in degrees instead of radians. See also [member global_rotation_degrees].
+			[b]Note:[/b] This is [b]not[/b] the property available in the Inspector dock.
 		</member>
 		<member name="scale" type="Vector2" setter="set_scale" getter="get_scale" default="Vector2(1, 1)">
-			The node's scale, relative to the node's parent. Unscaled value: [code](1, 1)[/code]. See also [member global_scale].
-			[b]Note:[/b] Negative X scales in 2D are not decomposable from the transformation matrix. Due to the way scale is represented with transformation matrices in Godot, negative scales on the X axis will be changed to negative scales on the Y axis and a rotation of 180 degrees when decomposed.
+			Local scale of this node in local space (relative to the parent node). See also [member global_scale].
+			[b]Note:[/b] The behavior of some 2D node types is not affected by this property. These include [Camera2D], [AudioStreamPlayer2D], etc.
+			[b]Warning:[/b] Mixed negative scales in 2D are not decomposable from the transformation matrix. Due to the way scale is represented with transformation matrices in Godot, the scale values will either be all positive or all negative.
 		</member>
 		<member name="skew" type="float" setter="set_skew" getter="get_skew" default="0.0">
-			If set to a non-zero value, slants the node in one direction or another. This can be used for pseudo-3D effects. See also [member global_skew].
-			[b]Note:[/b] Skew is performed on the X axis only, and [i]between[/i] rotation and scaling.
-			[b]Note:[/b] This property is edited in the inspector in degrees. If you want to use degrees in a script, use [code]skew = deg_to_rad(value_in_degrees)[/code].
+			Local skew applied to this node in the X axis, in radians and in local space (relative to the parent node). Higher values distort the node more. See also [member global_skew].
```

Comment:
No mention of `and [i]between[/i] rotation and scaling.` anymore. I can't say if it should be mentioned or not.